cif_rho.dic

```diff
@@ -690,13 +690,15 @@ save_atom_rho_multipole_coeff.list
     _definition.update            2019-04-01
     _description.text
 ;
-    Specifies the multipole population coefficients P(l,m) for
-    the atom identified in atom_rho_multipole_atom_label.  The
-    multipoles are defined with respect to the local axes specified
-    in the ATOM_LOCAL_AXES category.  The coefficients refer to the
-    multipole formalism described by Hansen & Coppens [1978,
-    equation (2)] which gives the electron density at position
-    vector r with respect to an atomic nucleus as
+    Specifies the multipole population coefficients P(l,m) for the
+    atom identified in atom_rho_multipole_atom_label. The list is
```

Comment:
```suggestion
    atom identified in _atom_rho_multipole.atom_label. The list is
```

Reply:
Agree. Also other instances where the all_underscore variant is used in a description. Will address all the issues raised here with a fresh commit.